### DISCUSSION TITLE
Draw Straightline or Constuction lines/Ruler?

### DISCUSSION THREAD

**Discussion by Piscabo** [0Tick/a1111-mini-paint#4] [Q&A]
Maybe I'm missing it, but I would love to be able to draw a straight line between 2 points, or maybe a ruler function. Or is it already there and I cant find it? Thanks. Great little addition to A1111.

**Comment by Piscabo** [comment]
I tried the edit function, but got this result?
![image](https://github.com/0Tick/a1111-mini-paint/assets/123719842/7011b802-7185-4e70-b1d0-76557a950505)

**Reply by 0Tick** [reply]
You need to go into the miniPaint Folder inside the extension folder, something like this `stable-diffusion-webui\extensions\a1111-mini-paint\miniPaint`. Then try running it again
